src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/presentation/NavigationButtons.tsx

```diff
@@ -59,13 +59,13 @@ export function NavigationButtons(props: INavigationButtons) {
   const OnClickNext = () => {
     const runPageValidations = !returnToView && triggers && triggers.includes(Triggers.ValidatePage);
     const runAllValidations = returnToView || (triggers && triggers.includes(Triggers.ValidateAllPages));
-    const validations = runAllValidations ? 'allPages' : (runPageValidations ? 'page' : null);
+    const runValidations = runAllValidations ? 'allPages' : (runPageValidations ? 'page' : null);
```

Comment:
Maybe we can avoid nested ternaries?
`(runAllValidation && 'allPages') || (runPageValidations && 'page');`

Reply:
Ah, neat!

Reply:
also, `'allPages'` and `'page'` seems like good candidates to be references to constants.
They are after all reused.